### PR TITLE
Syntax updates and testing shellcheck integration in Jenkins. 

### PR DESCRIPTION
**Pull request by pneerincx**

### CODE REVIEW

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

**Comment by pneerincx** [comment]
retest this please

File: bin/copyConcordanceCheckData.sh [hunk 1]
```diff
@@ -213,7 +220,7 @@ else
 		# Process this sample sheet / run.
 		#
 		filePrefix="$(basename "${sampleSheet%%.*}")"
```
Comment (RoanKanninga) [inline bin/copyConcordanceCheckData.sh]: This will only work when there are only dots in the filename and not in the path.

**Pull request merged**